docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -14,7 +14,7 @@ The `Estimator` primitive can compute an expectation value, and the `Sampler` pr
 Follow these instructions to get the expected value of an observable for a given quantum circuit with the [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) primitive.
 
 <Admonition type="note">
-  While this guide uses Qiskit’s reference implementation, the `Estimator` primitive can be run with any provider using [`qiskit.primitives.BackendEstimator`](../api/qiskit/qiskit.primitives.BackendEstimator).
+  While this guide uses the Qiskit reference implementation, the `Estimator` primitive can be run with any provider using [`qiskit.primitives.BackendEstimator`](../api/qiskit/qiskit.primitives.BackendEstimator).
 
   ```python
   from qiskit.primitives import BackendEstimator
```

Comment:
```python
from qiskit_ibm_runtime import QiskitRuntimeService EstimatorV2

service = QiskitRuntimeService()
estimator = EstimatorV2(backend=service.backend("backend_name"))
```

and I suppose this is actually an example of a provider that is an example of this: "There are some providers that implement primitives natively (see [the Qiskit Ecosystem page](https://qiskit.github.io/ecosystem/#primitives) for more details)." 

It's somewhat confusing because qiskit_ibm_runtime is not actually a provider proper, it just provides provider-like abilities. There is no generic backend _and_ primitive V2 compatible implementation, like in this existing example.

Reply:
Are you recommending that we change these examples to use Qiskit Runtime?

Reply:
I don't think I understand the goal of this document is well enough to answer that question confidently, because I've only looked at it briefly. I think the answer should be "yes" because qiskit_ibm_providers will be sunsetted and will never have a V2 implementation. Is this page supposed to talk about V1 and V2, or just V2?

Maybe @jyu00 can shed some light.

Reply:
The goal of this document is for people to test their quantum program (that calls primitives) locally, before sending them to Qiskit Runtime to run on a real QPU. 

Most of this page uses the statevector simulator, and there is an equivalent for V2, better named as [StatevectorEstimator](https://github.com/Qiskit/qiskit/blob/main/qiskit/primitives/statevector_estimator.py#L31) and [StatevectorSampler](https://github.com/Qiskit/qiskit/blob/main/qiskit/primitives/statevector_sampler.py#L52). 

In this `Note`, however, it talks about `BackendEstimator` which is not yet available for v2 (this is the [issue](https://github.ibm.com/IBM-Q-Software/ws-primitives/issues/130)). Similarly, there is currently no implementation in Aer for V2. If we somehow don't get those done prior to the release, I'd suggest adding a note saying they're "coming soon" for v2.

Reply:
This document was originally an introduction to primitives that talked about reference/backend/aer, but it has semi-evolved now to be a page on local simulation using the reference primitives. I think that the best action here is to remove the references to backend primitives and find them a new home, they don't fit here and in my opinion blur out the purpose of this page (I've made a couple of comments in this direction).